docs/templates/explanation-template.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Noun or nouns describing your subject
+content_type: explanation
+
+# Optional values. Uncomment any that apply to your document.
+
+# no_version: true # Disables the version selector dropdown. Set this on pages that belong to doc sets without versions like /konnect/.
+# alpha: true # Labels the page as alpha; adds a banner to the top of the page.
+# beta: true # Labels the page as beta; adds a banner to the top of the page.
+---
+
+Add an introduction paragraph here. Good introductions explain who this explanation topic is for and what this guide will cover.
```

Comment:
```suggestion
Add an introduction paragraph here. Good introductions explain who this explanation topic is for and what this guide will cover.
```
I think these intro paragraphs should fit the content type. So maybe "Good explanation documentation introductions express ..."

Reply:
With examples, even if they are made up.